dependencies/tekton-config/tekton-config.yml

```diff
@@ -42,3 +42,45 @@ spec:
                         readOnly: true
   pipeline:
     default-service-account: appstudio-pipeline
+    options:
+      disabled: false
+      deployments:
+        tekton-pipelines-controller:
+          spec:
+            template:
+              spec:
+                volumes:
+                  - name: trusted-ca
+                    configMap:
+                      name: trusted-ca
+                      items:
+                        - key: ca-bundle.crt
+                          path: ca-bundle.crt
+                      optional: true
+                containers:
+                  - name: tekton-pipelines-controller
+                    env:
+                    - name: SSL_CERT_FILE
+                      value: /etc/ssl/certs/ca-bundle.crt
```

Comment:
Do we need this env var? (do we need the cert in this pod at all?)

Reply:
fixed in https://github.com/konflux-ci/konflux-ci/pull/509/commits/2ced24e1f9faacce7bbf1d701099185c8910a5f4